Sources/MapboxDirections/DirectionsResult.swift

```diff
@@ -157,6 +157,13 @@ open class DirectionsResult: Codable {
      */
     open var routeIdentifier: String?
     
+    /**
+     Route index position as was provided by corresponding directions response.
+     
+     `Directions.calculate` methods may produce multiple alternative routes. `routeIndex` displays order of appearance in the response. This parameter might be useful for route refreshing.
+     */
+    open var routeIndex: Int?
```

Comment:
We’ve long tried to avoid introducing explicit index properties, instead relying on the containing structure (in this case an array in RouteResponse) to enforce order. If we need to introduce this property, I think we should make it constant to avoid confusion and possible scenarios where it gets out of sync with the actual order in the list of results.